airbyte-webapp/src/components/TextWithHTML/TextWithHTML.tsx

```diff
@@ -1,28 +1,31 @@
 import React from "react";
-import { FormattedHTMLMessage } from "react-intl";
 import sanitizeHtml from "sanitize-html";
 
 type IProps = {
   text?: string;
 };
 
+const allowedAttributes = {
+  ...sanitizeHtml.defaults.allowedAttributes,
+  a: [...sanitizeHtml.defaults.allowedAttributes["a"], "rel"],
+};
+
 const TextWithHTML: React.FC<IProps> = ({ text }) => {
   if (!text) {
     return null;
   }
 
   const sanitizedHtmlText = sanitizeHtml(text, {
+    allowedAttributes,
     transformTags: {
-      a: sanitizeHtml.simpleTransform("a", { target: "_blank" }),
+      a: sanitizeHtml.simpleTransform("a", {
+        target: "_blank",
+        rel: "noopener noreferrer",
```

Comment:
:information_source: Make sure all links are getting a proper `rel` tag to prevent the "opener attack" (see also https://mathiasbynens.github.io/rel-noopener/) in all browsers.